corpus/templates/pages/team.html

```diff
@@ -0,0 +1,399 @@
+  </head>
+
+  <div class="title">
+    <h1> Teams </h1>
+  </div>
+
+  <div class="center mt-10 align-center">
+    <div role="tablist" class="tabs tabs-boxed tabs-lg">
+
+      <a role="tab" id="IEEE-tab" class="tab" onclick="openPage('IEEE', this, 'IEEE-tab')">IEEE</a>
+
+      <a role="tab" id="CompSoc-tab" class="tab" onclick="openPage('CompSoc', this, 'CompSoc-tab')">CompSoc</a>
+
+      <a role="tab" id="Diode-tab" class="tab" onclick="openPage('Diode', this, 'Diode-tab')">Diode</a>
+
+      <a role="tab" id="Piston-tab" class="tab" onclick="openPage('Piston', this, 'Piston-tab')">Piston</a>
+
+    </div>
+
+      <div id="IEEE" role="tabpanel" class="tab-content bg-base-100 border-base-300 rounded-box p-6">
+        <h2 class="section-heading">IEEE Faculty</h2>
+        <br>
+        <div class="flex-container">
+          <!-- Faculty -->
+          {% for fac in faculty %}
+          {% if fac.post == "Branch Counselor" %}
+          <div class="margins">
+            <div class="card card-bordered w-96 bg-base-300 shadow-xl">
+              <figure class="px-10 pt-10">
+                <img class="profile-pic" {% if fac.profile_picture.url is not None %} src="{{fac.profile_picture.url}}" {% endif %} alt="photo" class="rounded-xl" />
+              </figure>
+              <div class="card-body items-center text-center">
+                <h2 class="card-title">{{fac.user}}</h2>
+                <p class="card-post">{{fac.post}}</p>
+                <br/>
+                <div class="card-actions">
+                  <button action="{{fac.user.executivemember.linkedin}}" target="_blank" class="btn btn-outline btn-lg"><i class="fab fa-2x fa-linkedin"></i></a>
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% endfor %}
+        </div>
+
+        <h2 class="section-heading">IEEE Core</h2>
+        <br>
+        <div class="flex-container">
+
+          <!-- Core -->
+          {% for core in ieee_core %}
+          <div class="margins">
+            <div class="card card-bordered w-96 bg-base-300 shadow-xl">
+              <figure class="px-10 pt-10">
+                <img class="profile-pic" {% if core.user.executivemember.profile_picture.url is not None %} src="{{core.user.executivemember.profile_picture.url}}" {% endif %} alt="photo" />
+              </figure>
+              <div class="card-body items-center text-center">
+                <h2 class="card-title">{{core.user}}</h2>
+                <p class="card-post">{{core.post}}</p>
+                <br/>
+                <div class="card-actions">
+                  <a href="{{core.user.executivemember.linkedin}}" target="_blank" class="btn btn-outline btn-lg"><i class="fab fa-2x fa-linkedin"></i></a>
+                  <a href="{{core.user.executivemember.github}}" target="blank" class="btn btn-lg btn-outline"><i class="fab fa-github fa-2x"></i></a>
```

Comment:
We need to check if these parameters are defined for the user. These fields aren't mandatory for an executive member, so some might have empty GitHub usernames and empty Linkedin URLs.